packages/data-point/lib/entity-types/entity-collection/factory.js

```diff
@@ -51,10 +51,16 @@ function validateComposeVsInlineModifiers (spec, invalidInlinesKeys) {
   }
 }
 
-function create (spec) {
+/**
+ * Creates new Entity Object
+ * @param  {Object} spec - spec
+ * @param {string} id - Entity id
+ * @return {Collection} Entity Object
```

Comment:
EntityCollection for consistency

Reply:
not in the scope of this issue

Reply:
but you're adding the "@return {Collection} Entity Object" line as part of this PR, so it's not a separate issue